### PR TITLE
build: add github action

### PR DESCRIPTION
**Pull request by helio-frota**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +53,7 @@ The `.nodeshift` directory contains your resource fragments.  These are `.yml` f
 
 OpenShift resource fragments are user provided YAML files which describe and enhance your deployed resources.  They are enriched with metadata, labels and more by nodeshift.
 
-Each resource gets its own file, which contains some skeleton of a resource description. Nodeshift will enrich it and then combine all the resources into a single openshift.yml and openshift.json(located in ./tmp/nodeshift/resource/).
+Each resource gets its own file, which contains some skeleton of a resource description. Nodeshift will enrich it and then combine all the resources into a single openshift.yml and OpenShift.json(located in ./tmp/nodeshift/resource/).
```
Comment (richardlau) [inline README.md:56]: Is this meant to have changed case?

File: package.json [hunk 2]
```diff
@@ -6,13 +6,9 @@
     "nodeshift": "./bin/nodeshift"
   },
   "scripts": {
-    "lint": "eslint bin/* lib/**/*.js test/*-test.js test/**/*-test.js index.js",
-    "test": "cross-env NODESHIFT_QUIET=true tape test/*-test.js test/**/*-test.js | tap-spec",
-    "coverage": "nyc npm test",
-    "coverage:html": "nyc npm test && nyc report --reporter=html",
```
Comment (lholmquist) [inline package.json:12]: can we keep this one.  I find it useful when i run things locally to see where i need coverage

File: README.md (hunk 1 above)
Comment (lholmquist) [reply README.md:56]: this is a file name, so it shouldn't change case

Comment (helio-frota) [reply README.md:56]: I can revert, that was due quick small change to trigger other build to avoid circle CI running job again.
Thanks for the review.

Comment (helio-frota) [reply README.md:56]: thanks for the review :+1:

File: package.json (hunk 2 above)
Comment (helio-frota) [reply package.json:12]: when we run `npm it` we already have lint + test + coverage

File: README.md (hunk 1 above)
Comment (helio-frota) [reply README.md]: @lholmquist @richardlau filename reverted :+1:

File: package.json (hunk 2 above)
Comment (lholmquist) [reply package.json:12]: yea,  i know,  but `coverage:html` outputs a nice html version that i look at when doing local development

**Review by richardlau — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (helio-frota) [reply package.json:12]: coverage html report added back :+1:

Comment (helio-frota) [reply package.json:12]: `npm run coverage`

**Review by lholmquist — APPROVED** [review]

**Pull request merged**